src/charactersheet/models/character/feat.js

```diff
@@ -1,28 +1,65 @@
 import { KOModel } from 'hypnos';
+import { Notifications } from 'charactersheet/utilities';
+import { Tracked } from './tracked';
+import { isEmpty } from 'lodash';
 import ko from 'knockout';
 
 export class Feat extends KOModel {
     static __skeys__ = ['core', 'feats'];
 
     static mapping = {
-        include: ['coreUuid', 'tracked']
+        include: ['coreUuid', 'tracked'],
+        tracked: {
+            update: ({ data, parent }) => {
+                const tracked = new Tracked();
+                if (!isEmpty(data) && !tracked.equals(data)) {
```

Comment:
A tracked class instance is not an empty js object, since it has built in functions. As long as we verify that the tracked data that is passed in is never a Tracked class object, then we can remove the second check.